### ISSUE TITLE
Funding Emulator accounts with FlowToken

### ISSUE THREAD

**Issue by sisyphusSmiling** [onflow/flow-cli#775]
### Issue To Be Solved

Funding emulator accounts with Flow isn't currently easy to do in the emulator.

### Suggest A Solution

One interface solution would be an attribute in the `flow.json`.

```
	"deployments": {
		"emulator": {
			"emulator-account": {
				"contracts": ["ContractA"],
				"initialBalance": 100.0
		}
	}
```

**Comment by sideninja** [comment]
I think funding an account should just be one of the templates for sending transactions. There is some planned work for that soon.

**Comment by sideninja** [comment]
Solved in v0.46.0
